test/Interop/Cxx/class/type-classification-module-interface.swift

```diff
@@ -24,6 +18,14 @@
 // CHECK-NOT: StructWithInheritedDeletedDestructor
 // CHECK-NOT: StructWithSubobjectDeletedDestructor
 
+// CHECK: struct StructWithMoveConstructor
+
+// CHECK: struct StructWithInheritedMoveConstructor
+
+// CHECK: struct StructWithSubobjectMoveConstructor
+
+// CHECK: struct StructNonCopyableTriviallyMovable
+
```

Comment:
Could you please bring back the checks for `StructWithPointerNonCopyableTriviallyMovable`
and `StructWithPointerNonCopyableTriviallyMovableField`?